src/Symfony/Component/Messenger/Bridge/Redis/Transport/Connection.php

```diff
@@ -120,8 +120,13 @@ public static function fromDsn(string $dsn, array $redisOptions = [], \Redis $re
     {
         $url = $dsn;
 
-        if (preg_match('#^redis:///([^:@])+$#', $dsn)) {
-            $url = str_replace('redis:', 'file:', $dsn);
+        $scheme = 'redis';
+        if (0 === strpos($dsn, 'rediss:')) {
```

Comment:
I suppose you have to change https://github.com/symfony/messenger/blob/5.x/Transport/TransportFactory.php#L46 too to get there, right?

Reply:
Thanks for the headsup! Changed it now :slightly_smiling_face: